pod-operation/src/components/ina219.rs

```diff
@@ -0,0 +1,10 @@
+use ina219::INA219;
+use rppal::i2c::I2c;
+use tracing::debug;
+
+pub fn read_current(ina: &mut INA219<I2c>) -> f32 {
```

Comment:
Suggestion: since the INA219 could have theoretically have varying configurations for different sensors (e.g. the pressure transducers operate at a given current range, but another device might require a different sampling range/resolution), I feel creating our own intermediate abstraction is a bit excessive unless we also decide to further parameterize this which then becomes more like implementing what the library should be doing. However, I do like how the calibration value and LSB scaling are more linked together although ideally the library itself should be handling that.